packages/components/src/components/iot-table/iot-table.tsx

```diff
@@ -23,6 +23,8 @@ export class IotTable {
 
   @Prop() queries!: TimeQuery<TimeSeriesData[], TimeSeriesDataRequest>[];
 
+  @Prop() staticDataStreams: SynchroChartsDataStream[];
```

Comment:
For my understanding: is this also for alarms? why "static"? can alarms be "queried" for, similar to time-series data?

Reply:
So the table component also displays static alarm alarm threshold data like rule.